pkgs/applications/networking/shellhub-agent/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "shellhub-agent";
+  version = "0.4.2";
+
```

Comment:
This should probably be a different commit:
```
shellhub-agent: init at 0.4.2
```

Reply:
Done.